address-controller/src/main/resources/swagger.json

```diff
@@ -0,0 +1,640 @@
+                        "schema": {
+                            "$ref": "#/definitions/AddressList"
+                        }
+                    }
+                ],
+                "responses": {
+                    "405": {
+                        "description": "Invalid input"
+                    }
+                }
+            },
+            "put": {
+                "tags": [
+                    "addresses"
+                ],
+                "summary": "Replace Addresses in an AddressSpace",
+                "description": "Replace Addresses in the specified AddressSpace with those specified in the request body.",
+                "operationId": "replaceAddresses",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "name": "addressSpace",
+                        "in": "path",
+                        "description": "Name of AddressSpace",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "in": "body",
+                        "name": "body",
+                        "description": "AddressList object",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/AddressList"
+                        }
+                    }
+                ],
+                "responses": {
+                    "405": {
+                        "description": "Invalid input"
+                    }
+                }
+            },
+            "delete": {
+                "tags": [
+                    "addresses"
+                ],
+                "summary": "Delete all Addresses in an AddressSpace",
+                "description": "",
+                "operationId": "deleteAllAddresses",
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "name": "addressSpace",
+                        "in": "path",
+                        "description": "Name of AddressSpace",
+                        "required": true,
+                        "type": "string"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Deletion successful; Empty AddressList is returned.",
+                        "schema": {
+                            "$ref": "#/definitions/AddressList"
+                        }
+                    },
+                    "404": {
+                        "description": "AddressSpace not found"
+                    }
+                }
+            }
+        },
+        "/addressses/{addressSpace}/{address}": {
+            "get": {
```

Comment:
I'm a little confused by this hierarchy.  I would have thought it would be something like.

AddressSpaces/{addressSpace}/addressess/address

Reply:
Yup, paths need to be cleaned up.